src/prted/prun_common.c

```diff
@@ -767,6 +767,7 @@ int prte_prun_parse_common_cli(void *jinfo, pmix_cli_result_t *results,
     prte_pmix_app_t *app;
     char *param;
     pmix_info_t info;
+    PRTE_HIDE_UNUSED_PARAMS(info);
```

Comment:
Again, just a problem of the use of "info" occurring within a clause (`PMIX_GPU_SUPPORT`) not defined within the PMIx version you used. I would suggest moving the declaration to line 900 so it sits within the clause. Hiding things that are actually used is probably not a good idea.